web-backend/src/main/java/com/statusneo/vms/controller/VisitorController.java

```diff
@@ -144,15 +152,13 @@ public ResponseEntity<String> validateOtp(
     public String searchEmployees(@RequestParam("employee") String query) {
         logger.info("Received search request for employee: {}", query);
         List<Employee> employees = employeeService.searchEmployeesByName(query);
-        StringBuilder html = new StringBuilder();
-        for (Employee employee : employees) {
-            html.append("<li class='px-3 py-2 hover:bg-gray-100 cursor-pointer' onclick='selectEmployee(\"")
-                    .append(employee.getName())
-                    .append("\")'>")
-                    .append(employee.getName())
-                    .append("</li>");
-        }
-        return html.toString();
+
```

Comment:
I don't think you need to do it this way. Do it the MVC way, get the MVC model and add your values to that, then return the template name. Spring MVC will do the rendering.